src/vlinder/trbs.py

```diff
@@ -41,6 +43,8 @@ def __init__(self, name, file_path=None, file_extension=None):
         self.output_dict = {}
         self.visualizer = None
         self.exporter = None
+        self.report = None
+        self.random = random.random()
```

Comment:
Why store the random number here? And not within the Visualizer-class?

Reply:
Because you need the same random number for visualize as for make_report